guides/common/modules/proc_increasing-the-logging-levels-to-help-with-debugging.adoc

```diff
@@ -33,7 +33,7 @@ ifndef::foreman-deb[]
 COMPONENT  | AUTO-DETECTED BY EXISTENCE OF       | DESTINATIONS
 -----------|-------------------------------------|-------------------------------------
 dhcpd      | /etc/dhcp/dhcpd.conf                | syslog /var/log/dhcpd-debug.log
-postgresql | /var/lib/pgsql/data/postgresql.conf | syslog /var/lib/pgsql/data/pg_log/
```

Comment:
Now I'm wondering what to do. It looks like like the location is hardcoded and so is the destination:
https://github.com/theforeman/hammer-cli-foreman-admin/blob/31e7937c8392db34803f7324feb506d7146e19de/config/cli.modules.d/foreman_admin_logging_core.yml#L13

That said, `/var/lib/pgsql/data/pg_log/` doesn't appear to exist. I see in the fixtures it may have been the `log_directory` in the past (https://github.com/theforeman/hammer-cli-foreman-admin/blob/31e7937c8392db34803f7324feb506d7146e19de/test/fixture_tree/var/lib/pgsql/data/postgresql.conf#L312), but on my installation it's just plain `log` instead of `pg_log`.

I'm leaning to reverting this line and leaving a comment in it that the locations are hardcoded and shouldn't use macros.

Given it was never updated for SCL, I wonder how many people actually use it anyway. It's not packaged for Debian anyway. Also, there are no instructions to install it and on non-Satellite nothing pulls it in either.

Reply:
No idea where this is coming from. The upstream example file also uses `log`: https://github.com/postgres/postgres/blob/master/src/backend/utils/misc/postgresql.conf.sample#L457

There used to be a `pg_log` *file*, but that's gone for, uh, a while: https://github.com/postgres/postgres/commit/2589735da08c4e597accb6eab5ae65b6339ee630

I'd revert that particular hunk and make an issue against hammer-admin to fix this up correctly.